### DISCUSSION TITLE
auto save

### DISCUSSION THREAD

**Discussion by MAMware** [gsantner/markor#1686] [Ideas] [answered]
Hello, it happens to me more often that whatsnot that the app closes it self suddently with no error display, no warning no nothing. i could send some kind of log if there is such thing but... in the mean time.

is it possible to have an auto save feature? like ever 10 seconds?

**Comment by gsantner** [comment, marked as answer]
generaly markor auto saves when you leave the app or the document. if thats not working, then this issue should be fixed.   
for manual saving, there is the save button.  
automatically every X seconds saving produces unwanted file modification timestamps, not something we want.

**Reply by MAMware** [reply]
hi! markor was closing be it self (im a samsung DeX user), maybe some kind of crash due to that so it didnt save anything. anyway i havent experienced such crash for quite a while now. thank you for the reply
